plugin.yaml

```diff
@@ -42,10 +37,6 @@ node_types:
           https://cloud.google.com/compute/docs/machine-types
         type: string
         default: n1-standard-1
-      instance_additional_settings:
-        description: >
-          Additional instance settings.
-        default: {}
```

Comment:
Ah, yes I remember we discussed this, but then I couldn't find/remember where you'd added it (I guess I was looking at the cloudify-cosmo project by mistake). I'll add this back in.